### ISSUE TITLE
enum 

### ISSUE THREAD

**Issue by kpgarrod** [TypeStrong/atom-typescript#282]
I have two enums defined in two different files in the same project:

file1.ts:  enum ```WebSitePageStatusEnum { Hidden, Shown, AlwaysShown, AlwaysHidden }```
file2.ts:  enum ```PlanTypeEnum { Free, Bronze, Silver, Gold, Platinum }```

In file3.ts, I want to use both enums, so I add:

>enum PlanTypeEnum { }
enum WebSitePageStatusEnum { }

Now I get autocompletion for PlanTypeEnum but not for WebSitePageStatusEnum. Also, when I try to use say ```WebSitePageStatusEnum. Shown``` in my code, I get an error: ```"Property 'Shown' does not exist on type 'typeof WebSitePageStatusEnum'".```

I can't see any significant difference between the generated .js files for file1.ts and file2.ts or in the code generated in file3.ts for the two enums. The code executes correctly.

What am I missing please?

**Comment by poelstra** [comment]
You're basically extending/redefining your enums in file3.ts. I don't think that's what you want.
Are you using external modules? If so, you should probably:
```ts
/// file2.ts
export enum PlanTypeEnum { Free, Bronze, etc }

/// file3.ts
import { PlanTypeEnum } from "./file2.ts"
var foo = PlanTypeEnum.Free;
```

**Comment by kpgarrod** [comment]
@poelstra: thanks. I tried to use external modules but was having difficulty define a path to the file that didn't generate an error!

I think I had some conflict between autocomplete plugins. I've disabled everything except atom-typescript and now have export/import working as you described. Thanks for the help.

**Comment by basarat** [comment]
> I have been trying to use external modules but was having difficulty define a path to the file that didn't generate an error

Atom-ts should autocomplete the correct path for you. Please see the example

**Comment by kpgarrod** [comment]
Thanks for the support. Atom-ts does indeed autocomplete the path once autocomplete-path is out of the way. My Typescript journey is back on track!
